NSubstitute.DbConnection/NSubstitute.DbConnection/MockQuery.cs

```diff
@@ -71,6 +71,12 @@ public IMockQueryResultBuilder Returns<T>(Func<QueryInfo, IEnumerable<T>> result
 
         public void Affects(Func<QueryInfo, int> rowCountSelector) => RowCountSelector = rowCountSelector;
 
+        public void Throws<T>(T exception)
+            where T : Exception
+        {
```

Comment:
You can use an expression body here, also the parens around the discard and the braces around the expression body are redundant

Reply:
the expression body does't play well with [SA1127](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1127.md)
```csharp
# StyleCop complains SA1127 here
public void Throws<T>(T exception) where T : Exception => ResultSelectors.Add((typeof(T), _ => { throw exception; }));
```

```csharp
# StyleCop is happy but it looks weird to me
# Also it doesn't play well with the Format Document feature in Visual Studio
public void Throws<T>(T exception)
    where T : Exception => ResultSelectors.Add((typeof(T), _ => { throw exception; }));
```
so at the end of the day I went with a "normal" method instead of expression body but let me know if you prefer any other way.